parlai/tasks/eli5/data_creation/slurm_scripts/eli_download_docs.sbatch

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
nit: should filename be `eli5_download_docs.sh` instead?

Reply:
I like sbatch, it emphasizes that this is best on slurm.